tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+	"compilerOptions": {
+		"module": "Node16",
+		"moduleResolution": "Node16",
+		"target": "ES2022",
+		"lib": [
+            "ES2022"
+        ],
+		"outDir": "out",
+		"resolveJsonModule": true,
+		"strict": true,
+		"forceConsistentCasingInFileNames": true,
+		"skipLibCheck": true, // don't know how to fix it
```

Comment:
what is the error when not skipping it?

Reply:
should be fixed with https://github.com/redhat-developer/vscode-extension-tester/pull/1521

Reply:
either we wait for a release with the fix or create an issue to remember to change that

Reply:
I will do release with latest VS Code support anyway so I will wait for it, and hopefully the `skipLibCheck` will be removed then

Reply:
removed - fixed by latest extester release